src/Object/Contact.php

```diff
@@ -383,7 +384,7 @@ public static function photoMenu(array $contact, $uid = 0)
 
 			$r = dba::select('contact', array(), array('nurl' => $contact['nurl'], 'network' => $contact['network'], 'uid' => $uid));
 			if ($r) {
-				return self::photoMenu($r[0], $uid);
+				return self::photoMenu(dba::fetch($r), $uid);
```

Comment:
This looks like the query is expected to return only one row. So a ```array('limit' => 1)``` should be the best, I guess.